### DISCUSSION TITLE
Have any recommend plugin? (DisableMeteor, DisableMobspawn)

### DISCUSSION THREAD

**Discussion by BabeElena** [Pryaxis/TShock#1715] [General]
I finding for a plugin make you can disable meteor and mob spawn

Can someone recommend plugin to me?

(I can't use NPCFilter, FU Meteorite due too old version)
Sorry for grammar and thank you so much

**Comment by bippity** [comment]
[Event Stopper](https://tshock.co/xf/index.php?resources/event-stopper.64/) would be the plugin you're looking for. I've gone ahead and updated it for you to download [**here**](https://github.com/bippity/EventStopper/tree/master/EventStopper/bin/Release).

As for disabling mob spawn, that can be done using the `/maxspawns` command (`/maxspawns 0`). If you want to change it permanently, you just need to set `"DefaultMaximumSpawns": 5` to 0 in `tshock/config.json`.

**Reply by BabeElena** [reply]
Thank you so much!!
